system/jlib/jsocket.cpp

```diff
@@ -4846,6 +4850,8 @@ class CSocketEpollThread: public CSocketBaseThread
                         try
                         {
                             si.nfy->notifySelected(si.sock,si.mode); // ignore return
+                            si.nfy->Release();
+                            si.sock->Release();
```

Comment:
In other places, there's a try/catch around releasing a socket, in some places around both the nfy and the sock releases.
If necessary in other places, presumable should be here too.